### ISSUE TITLE
Accessibility issue for `Icon` component. `aria-busy="true"` should be used when icon is loading, and clear aria attribute when it's loaded

### ISSUE THREAD

**Issue by k-paxian** [Vonage/vivid-3#1678]
### Description

Reported by "AXE Accessibility" Results

`critical` `aria-required-children`  WCAG 2 Level A, WCAG 1.3.1
Ensures elements with an ARIA role that require child roles contain them

![image](https://github.com/Vonage/vivid-3/assets/18071010/d2294162-0160-4e99-91a2-f308226f8bb0)

### Component(s) impacted

Icon

### Vivid Web Components version

3.53.0

### Severity

Severity 1 = The design is broken in a critical way that blocks users from completing tasks or damages the brand. Affects major functionality, no workaround.

### Application/website

Agent Workspace

### CodeSandbox example

https://github.com/Vonage/vivid-3/blob/37d5e687cca8e42f01deb828acb35093d14fae84/libs/components/src/lib/icon/icon.template.ts#L18-L33

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/vonage/vivid-3/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/vonage/vivid-3/issues) for duplicate issues

**Comment by rachelbt** [comment]
@k-paxian 
I'm closing this issue - moved it to Jira:
https://jira.vonage.com/browse/VIV-1749
